roles/znc/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+znc_user: znc-admin
+znc_group: znc
+znc_port: 5000
+znc_ipv4: True
+znc_ipv6: False
+znc_ssl: True
+znc_data_dir: /var/lib/znc
+znc_config_dir: "/home/{{ znc_user }}/.znc/configs"
+
+znc_user_name: eggshell
+znc_user_password: <redacted> 
```

Comment:
need a good way to keep passwords out of plaintext. @mlangbehn what are your thoughts?